routes/authors.js

```diff
@@ -1,8 +1,55 @@
 const express = require("express");
 const generateId = require("uuid/v4");
+const authors = require("../data/authors");
+const body = require('body-parser')
+const books = require("../data/books");
+
+
 
 const router = express.Router();
 
 // Define your endpoints here
 
-module.exports = router;
+router.get('/', (req, res, next) => {
+	const sorter = req.query.sort
+	if (sorter) res.json(authors.map(el => el[sorter]).sort())
```

Comment:
If you do that you are only going to send the sorted names, not the authors sorted by names. Still good code however.

Reply:
Indeed, but that was on purpose